Is there any possibility that pygrib can be used on windows system without relying on cygwin environment?

Not working with anaconda + windows....anyone had any luck? Would be nice if I can support my software on windows.

```
[pyrate] C:\Users\sudipta>pip install git+https://github.com/jswhit/pygrib
Collecting git+https://github.com/jswhit/pygrib
  Cloning https://github.com/jswhit/pygrib to c:\users\sudipta\appdata\local\tem
p\pip-yia0qd-build
Installing collected packages: pygrib
  Running setup.py install for pygrib ... error
    Complete output from command C:\Users\sudipta\Anaconda2\envs\pyrate\python.e
xe -u -c "import setuptools, tokenize;__file__='c:\\users\\sudipta\\appdata\\loc
al\\temp\\pip-yia0qd-build\\setup.py';exec(compile(getattr(tokenize, 'open', ope
n)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record
c:\users\sudipta\appdata\local\temp\pip-1xrx7x-record\install-record.txt --singl
e-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-2.7
    copying ncepgrib2.py -> build\lib.win-amd64-2.7
    running build_ext
    building 'pygrib' extension
    creating build\temp.win-amd64-2.7
    creating build\temp.win-amd64-2.7\Release
    C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\BIN\amd64\cl.exe /c /n
ologo /Ox /MD /W3 /GS- /DNDEBUG -IC:\Users\sudipta\Anaconda2\envs\pyrate\lib\sit
e-packages\numpy\core\include -Ig2clib_src -IC:\Users\sudipta\Anaconda2\envs\pyr
ate\include -IC:\Users\sudipta\Anaconda2\envs\pyrate\PC /Tcpygrib.c /Fobuild\tem
p.win-amd64-2.7\Release\pygrib.obj
    pygrib.c
    c:\users\sudipta\anaconda2\envs\pyrate\lib\site-packages\numpy\core\include\
numpy\npy_1_7_deprecated_api.h(12) : Warning Msg: Using deprecated NumPy API, di
sable it by #defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
    pygrib.c(242) : fatal error C1083: Cannot open include file: 'grib_api.h': N
o such file or directory
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 9.0\\VC\\BI
N\\amd64\\cl.exe' failed with exit status 2

    ----------------------------------------
Command "C:\Users\sudipta\Anaconda2\envs\pyrate\python.exe -u -c "import setupto
ols, tokenize;__file__='c:\\users\\sudipta\\appdata\\local\\temp\\pip-yia0qd-bui
ld\\setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().repl
ace('\r\n', '\n'), __file__, 'exec'))" install --record c:\users\sudipta\appdata
\local\temp\pip-1xrx7x-record\install-record.txt --single-version-externally-man
aged --compile" failed with error code 1 in c:\users\sudipta\appdata\local\temp\
pip-yia0qd-build\
```

@jdkloe  Currently tried to do it but got this error message. Anyone knows how to fix it ?

`CMake Warning at cmake/ecbuild_log.cmake:153 (message):
  WARN - ENABLE_LARGE_FILE_SUPPORT active, sizeof off_t is 4 < 8 but ecbuild
  does not know how to enable large files in this operating system
Call Stack (most recent call first):
  cmake/ecbuild_add_large_file_support.cmake:25 (ecbuild_warn)
  cmake/ecbuild_check_os.cmake:55 (ecbuild_add_large_file_support)
  cmake/ecbuild_system.cmake:236 (include)
  CMakeLists.txt:25 (include)


CMake Error at cmake/ecbuild_log.cmake:191 (message):
  CRITICAL - ecBuild is untested for this operating system: [Windows] --
  refusing to continue.  Disable this check with -DDISABLE_OS_CHECK=ON
Call Stack (most recent call first):
  cmake/ecbuild_check_os.cmake:383 (ecbuild_critical)
  cmake/ecbuild_system.cmake:236 (include)
  CMakeLists.txt:25 (include)`

Adding a boolean variable in my Cache did not work. [noob] in case you didn't figure it out so far. Any help is appreciated.

--- a/setup.py
+++ b/setup.py
@@ -157,13 +157,15 @@ if hasattr(sys,'maxsize'):
 else:
     if sys.maxint > 2**31-1: macros.append(('__64BIT__',1))
 
+runtime_libdirs = libdirs if os.name != "nt" else None
 g2clibext = Extension("g2clib",g2clib_deps,include_dirs=incdirs,\
-            library_dirs=libdirs,libraries=libraries,runtime_library_dirs=libdirs,define_macros=macros)
+            library_dirs=libdirs,libraries=libraries,runtime_library_dirs=runtime_libdirs,
+            define_macros=macros)
 redtoregext =\
 Extension("redtoreg",[redtoreg_pyx],include_dirs=[numpy.get_include()])
 pygribext =\
 Extension("pygrib",[pygrib_pyx],include_dirs=incdirs,library_dirs=libdirs,\
-          runtime_library_dirs=libdirs,libraries=libraries)
+          runtime_library_dirs=runtime_libdirs,libraries=libraries)
 
 # man pages installed in man_dir/man1
 if man_dir is not None:

